### PR TITLE
Verify if weighted back-end pod selection is working properly

### PR DESCRIPTION
**Pull request by ESWZY**
Ref #467 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: eBPF_Supermarket/TrafficManager/acceptance/weigh_test.go [hunk 1]
```diff
@@ -0,0 +1,182 @@
+package acceptance
```
Comment (helight) [inline eBPF_Supermarket/TrafficManager/acceptance/weigh_test.go:17]: 文件头没有添加

File: eBPF_Supermarket/TrafficManager/main.go [hunk 2]
```diff
@@ -1,25 +1,10 @@
...
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-//
-// author: Woa <me@wuzy.cn>
-
 package main
 
 import (
 	"github.com/eswzy/eTrafficManager/bpf"
```
Comment (helight) [inline eBPF_Supermarket/TrafficManager/main.go:20]: 文件头怎么删除了

**Review by chenamy2017 — DISMISSED** [review]

File: eBPF_Supermarket/TrafficManager/main.go (hunk 2 above)
Comment (ESWZY) [reply eBPF_Supermarket/TrafficManager/main.go:20]: 补好了

**Review by LinkinPF — APPROVED** [review]

**Comment by LinkinPF** [comment]
后面可以对.o文件做一个说明记录

**Comment by ESWZY** [comment]
> 后面可以对.o文件做一个说明记录

好，那我下次在readme里介绍工程文件含义的时候加上

**Review by helight — APPROVED** [review]
LGTM

**Pull request merged**